types/application.yaml

```
@@ -16683,6 +16683,14 @@
     template:
     - application/zip
   registered: true
+- !ruby/object:MIME::Type
+  content-type: application/x-compressed-zip
```

Just wanted to confirm that the mime type should be `application/x-zip-compressed` (it's flipped in your actual code change)
```suggestion
  content-type: application/x-zip-compressed
```